vespalib/src/vespa/vespalib/util/mmap_file_allocator.cpp

```diff
@@ -9,10 +10,10 @@ namespace vespalib::alloc {
 
 namespace {
 
-const size_t page_size = std::max(getpagesize(), 4096);
+const size_t mmap_alignment = getpagesize();
 
-size_t round_to_page_size(size_t size) {
-    return (size + ((-size) & (page_size - 1)));
+size_t round_to_mmap_alignment(size_t size) {
```

Comment:
I still think we should make a utility function for this and use the same in alloc.cpp.